src/js/panels.js

```diff
@@ -107,7 +117,6 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
         var step = reverse ? (that.options.step * -1) : that.options.step;
         var newValue = that.model.value + step;
-        newValue = gpii.firstDiscovery.panel.ranged.clip(newValue, that.options.range.min, that.options.range.max);
```

Comment:
Can we use the "fluid.transforms.limitRange" transformation for implementing gpii.firstDiscovery.panel.ranged.clip() as well. Even better if one common member option can be shared for defining the transformation rule.